vm/native/gas.go

```diff
@@ -203,11 +211,11 @@ var (
 			}
 			contractName, cost0 := h.ContractName()
 			cost.AddAssign(cost0)
-			//cost0, err = h.Withdraw(userName, unpledgeAmountStr)
-			_, cost0, err = h.CallWithAuth("iost.token", "transfer", fmt.Sprintf(`["iost", "%v", "%v", "%v"]`, contractName, receiver, unpledgeAmountStr))
+			freezeTime := h.Context().Value("time").(int64) + UnpledgeFreezeDuration
```

Comment:
Out of curious, what does `freezeTime` means ? :)

Reply:
the moment when the tokens will be unfreezed